pkg/cross-compilers/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM lfedge/eve-alpine:1fc941603ddc153f26104865db242e644c16aa99 as build-amd64
```

Comment:
Don't we need --platform=${BUILDPLATFORM} here? It's building cross compiler that should run on the host, I'm wondering if some do a make ZARCH=arm64 ... pkg/cross-compilers on a x86 host... it will use emulation in this case...

Reply:
You are right, we need cross-compilers for the current platform we are sitting on (BUILDPLATFORM). But at the same time we want to have images pushed for all platforms in our CI. So we are forced to build them as we can (including emulation as we do not have riscv64 runner available). That is, when we run `make ZARCH=riscv64 pkgs` in CI we expect that it will construct riscv64-based image for us. The idea here is to not rebuild cross-compilers often.

Reply:
@giggsoff , ok, got it.